### DISCUSSION TITLE
3.0.0-beta17

### DISCUSSION THREAD

**Discussion by Him188** [open-ani/animeko#217] [Announcements]
### 新特性
- 支持发送弹幕
   > 会发送到 Ani 的弹幕服务器, 非中国大陆用户在设置中可以启用加速线路
   > 发送的弹幕与 bangumi 用户 ID 绑定
### 优化和修复
- 当视频已经缓存完成时, 使用本地文件播放, 大幅加快载入速度
- 修复视频总是要加载很久的问题 #214
- 修复删除缓存无效的问题
- 加快获取弹幕的速度 #216
- 优化自动更新逻辑, 现在每小时最多检查一次

----

**附件:**

- `.zip` 是包含 JRE 的解压即用的绿色版本；
- `.dmg`、`.deb`、`.rpm` 和 `.msi` 分别是 macOS、Debian、RedHat 和 Windows 平台的安装程序；
- `.exe` 是包含 JRE 的 Windows 程序，可以直接运行。
- `.jar` 是不包含 JRE 的最小程序，需要配合 JRE 11 及以上使用；
- `.apk` 是 Android 应用，支持的最低 Android 版本为 8.0；

注意，桌面平台仅支持 64 位操作系统。在使用 macOS 的安装程序时可能提示文件损坏，这是因为此版本应用没有签名，可以忽略错误继续使用。

也可扫描以下二维码下载 Android 版本：
![](https://github.com/Him188/ani/releases/download/v3.0.0-beta17/ani-3.0.0-beta17.apk.qrcode.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/open-ani/ani/releases/tag/v3.0.0-beta17'>3.0.0-beta17</a>.</em>

**Comment by NeKoOuO** [comment]
可以用TG別用QQ群嗎🥹

**Reply by Him188** [reply]
可以都有

**Reply by Him188** [reply]
在看会有多少人加群

**Reply by Him188** [reply]
https://t.me/+afDaMsCSIcsyNDVk

**Comment by NeKoOuO** [comment]
恭喜突破300🎉可喜可賀~看來宣傳有效

**Comment by NeKoOuO** [comment]
如果下載集數（不是播放）是那種1-12的合輯是指會下載其中選的那集嗎？ 還是說有辦法按一次全下載

**Reply by Him188** [reply]
只会下载那一集。没有办法全下载目前。

**Reply by NeKoOuO** [reply]
> 只会下载那一集。没有办法全下载目前。

感覺要有個一次全的 不然要重複動作12次
